_data/locales/ja.yml

```
@@ -0,0 +1,31 @@
+ja:
+  locale_name: 日本語
+  nav:
+    about: このサイトについて
+    contribute: 貢献する
+  index:
+    lead: オープンソースソフトウェアはあなたのような人々によって作られています。プロジェクトを立ち上げて大きくしていく方法を学びましょう。
```

```suggestion
    lead: オープンソースソフトウェアはちょうどあなたのような人々によって作られています。プロジェクトを立ち上げて成長させていく方法を学んでいきましょう。
```
語りかけるようなニュアンスにしてみました